.github/workflows/main.yml

```diff
@@ -15,7 +18,10 @@ jobs:
         - ubuntu-latest
         - macos-latest
         - windows-latest
+      fail-fast: false
```

Comment:
I'm guessing this change is an artifact of working through the changes? Is there a reason to keep this change going forward?

Reply:
In my test, some of the `pypy` tests failed early for unrelated reasons and immediately canceled the runs that I was interested in.

I would suggest to use `fail-fast: false` - all test runs are pretty short, so it's unlikely that this is problematic

Reply:
I'd prefer to avoid introducing variance from best practices where possible, and I presume fail-fast is a best practice as it's the default and the behavior deployed in all my other repos. So I'll omit it for now. If there's a good reason to introduce it, let's do that separately.

Reply:
Fixed in ec1c3f34c.